src/schema/vacationRequest.ts

```diff
@@ -1,15 +1,26 @@
 import { Type } from "@sinclair/typebox";
+import {VacationRequestStatuses} from "@generated/client/model/vacationRequestStatuses";
 
 /**
- * Schema for the Questionnaire table
+ * Schema for each individual status entry in a vacation request
  */
+const vacationRequestStatusSchema = Type.Object({
+  status: Type.Enum(VacationRequestStatuses),
+  message: Type.String(),
+  createdBy: Type.String(),
+  updatedAt: Type.String()
+})
 
+/**
+ * Schema for the vacation request table
+ */
 const vacationRequestSchema = Type.Object({
```

Comment:
This should match the spec e.g. draft is optional (or maybe it doesn't need to be optional), but I think this is missing at least personId? And status needs to be added to the spec. 

And just to be clear we don't want id here, and it's not here, so that fine!